src/aat/resources/features/F-047/S-574.td.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "must return 200 and a list of case ids a user has access to",
```

Comment:
Title is wrong . This is a negative scenario. Please update as per feature file.

Reply:
done